core/trace/src/main/scala-3/org/typelevel/otel4s/trace/TracerMacro.scala

```diff
@@ -59,7 +58,7 @@ private[otel4s] trait TracerMacro[F[_]] {
       inline name: String,
       inline attributes: Attribute[_]*
   ): SpanOps[F] =
-    ${ TracerMacro.span('self, 'name, 'attributes) }
+    spanBuilder(name).addAttributes(attributes).build
```

Comment:
I love how the only sense in which this is even a macro now is that everything is `inline`

Reply:
Yes, exactly. Scala 3 macro feels much better now.